pod/perldelta.pod

```diff
@@ -91,16 +91,12 @@ as an updated module in the L</Modules and Pragmata> section.
 
 =head1 Performance Enhancements
 
-XXX Changes which enhance performance without changing behaviour go here.
-There may well be none in a stable release.
-
-[ List each enhancement as an =item entry ]
-
```

Comment:
Usually we leave these bits in for individual updates. Deleting that chunk is part of the release process.

Reply:
Ok, restored it